config/prod.js

```diff
@@ -1,16 +1,15 @@
-const webpack=require('webpack');
-const { dir, common }=require('./base');
-const merge=require('webpack-merge');
-
-module.exports=function(env) {
-    return merge(common, {
-        plugins:[
-            new webpack.DefinePlugin({
-                'process.env': {
-                    'NODE_ENV': JSON.stringify('production')
-                }
-            }),
-
-        ],
-    });
-}
+const webpack = require('webpack');
+const { common } = require('./base');
+const merge = require('webpack-merge');
+
+module.exports = function() {
+  return merge(common, {
+    plugins: [
```

Comment:
I agree. However, in dev there is it's own separate entry point. But prod doesn't. So that we can add entry point to base.config, then both dev and prod will be merged with base.config.
This way we'll have dev with it's own entry and also prod uses entry from base and it works.
Because for now yarn build doesn't work at all.